docs-source/conf.py

```diff
@@ -0,0 +1,99 @@
+import datetime
+import toml
+import os
+
+project_root = os.path.dirname(os.path.dirname(__file__))
+project_metadata = toml.load(
+    os.path.join(project_root, 'pyproject.toml')
+)['tool']['poetry']
+
+# -- General configuration ------------------------------------------------
+
+# If your documentation needs a minimal Sphinx version, state it here.
+#
+# needs_sphinx = '1.0'
+
+# Add any Sphinx extension module names here, as strings. They can be
+# extensions coming with Sphinx (named 'sphinx.ext.*') or your custom
+# ones.
+extensions = [
+    'sphinx.ext.autodoc',
+]
+
+# Add any paths that contain templates here, relative to this directory.
+templates_path = ['_templates']
+
+# The suffix(es) of source filenames.
+# You can specify multiple suffix as a list of string:
+#
+# source_suffix = ['.rst', '.md']
+source_suffix = '.rst'
+
+# The master toctree document.
+master_doc = 'index'
+
+# General information about the project.
+author = ", ".join(project_metadata['authors'])
```

Comment:
Shouldn't this be `Omnivector Solutions`?

Reply:
I have the feeling `authors` are real people, not the company, but I'd like to keep that discussion

Reply:
It pulls that info from pyproject.toml where the authors list is `["Omnivector Solutions <info@omnivector.solutions>"]`

Reply:
Does sphinx understand this format? `bla <ble@bli.com>`?

Reply:
The author is just rendered as a string. However, I changed the conf.py to make it a hyperlink.